key-revocation.md

```diff
@@ -0,0 +1,75 @@
+# Key Revocation
+
+One of the goals of Notary v2 is to build in solutions for key revocation that are easy to use and ensure that users will always use non-compromised keys. This document discusses some potential mechanisms for key revocation.
+
+In existing systems, there are three main approaches to key revocation: automatic revocation through key expiration, key revocation lists, and providing a list of trusted keys. This document discusses some of the benefits and pitfalls of each of these techniques, and how some of these techniques are combined to provide a holistic approach to key revocation in TUF.
+
+
+## Key Expiration
+
+Adding an expiration time to every key allows keys to automatically be revoked after a certain period of time. The expiration time is usually included with the key so that it is easy for users to find. This technique does not require any action from the key holder, and ensures that users will have to refresh their trusted keys before those keys expire.
+
+However, if a key is compromised before it expires, key expiration alone does not protect users. An attacker with the compromised key could sign arbitrary images or metadata until the key's expiration time.
+
+Pros:
+* No action required from key holder
+* Simple: only requires a timestamp
+
+Cons:
+* Keys can't be revoked before expiration
+* Artifacts must be re-signed after expiration
```

Comment:
Timestamping is an option that allows use of signed artifacts after the key expires.